envisage/ui/action/abstract_action_manager_builder.py

```diff
@@ -68,7 +68,7 @@ def create_tool_bar_managers(self, root):
             groups = []
             for group in self._action_set_manager.get_groups(root):
                 if group.path.startswith("%s/%s" % (root, tool_bar.name)):
-                    group.path = "/".join(action.path.split("/")[1:])
+                    group.path = "/".join(group.path.split("/")[1:])
```

Comment:
im guessing this was a bug.

Reply:
Yes; for this one at least, the fix seemed obvious. For the name errors in the single project, not so much.

Reply:
And of course it should have a unit test, but not as part of this PR. I'll open an issue.

Reply:
Opened #273.